rootston/cursor.c

```diff
@@ -310,6 +310,9 @@ void roots_cursor_handle_motion(struct roots_cursor *cursor,
 	double dx = event->delta_x;
 	double dy = event->delta_y;
 
+	wlr_seat_pointer_notify_relative_motion(cursor->seat->seat, (uint64_t)
```

Comment:
nit: put the cast next to the statement it's casting, don't split the line here

Reply:
I wonder if these calls could live in wlr_cursor?

Reply:
Can't opine on that yet; not too familiar with the pipeline. It has to be done after pointer focus is calculated, at least.

Reply:
I checked; the analogous `wlr_seat_pointer_notify_motion` is called only in rootston. Same treatment for this one.